docs/content/docs/KnownIssues.md

```diff
@@ -11,3 +11,14 @@ type: docs
 Disable **Settings > Experimental > Only sync the active variant**
 
 https://stackoverflow.com/questions/54201216/android-studio-3-3-new-gradle-sync-is-not-supported-due-to-containing-kotlin-mo/
+
+## Tests run from Android Studio are failed in runtime with `java.lang.NoClassDefFoundError`
```

Comment:
"in runtime" is not clear for me. 
I think a run configuration is always running something.
My proposal: skip it.

"... failed with `java.lang.NoClassDefFoundError`"

Reply:
Fixed